components/BookmarkButton.js

```diff
@@ -7,19 +7,24 @@ export default function BookmarkButton({ toggleBookmark, medication }) {
     justify-self: flex-end;
     border: none;
     background-color: transparent;
-    .bookmarked {
-      background-color: goldenrod;
-    }
   `;
   return (
     <Button onClick={() => toggleBookmark(medication.id)}>
-      <Image
-        src="/bookmark.svg"
-        width={23}
-        height={23}
-        alt="bookmark"
-        className={medication.isBookmarked ? "bookmarked" : ""}
-      />
+      {medication.isBookmarked ? (
+        <Image
+          src={"/redBookmark.svg"}
+          alt="bookmarked"
+          width={25}
+          height={25}
+        ></Image>
+      ) : (
+        <Image
+          src={"/bookmark.svg"}
+          alt="not bookmarked"
+          width={25}
+          height={25}
+        ></Image>
+      )}
```

Comment:
A more minimal version of this could be:

```suggestion
      <Image
        src={medication.isBookmarked ? "/redBookmark.svg" : "/bookmark.svg"}
        width={25}
        height={25}
        alt=""
      />
```

This also removes the alt text. I'd suggest adding an `aria-label` to the surrounding button with the action text:

```jsx
<Button
  onClick={() => toggleBookmark(medication.id)}
  aria-label={medication.isBookmarked ? "remove bookmark" : "bookmark"}
>

```

Reply:
thank you for your feedback I`ll change it now